Cargo.toml

```diff
@@ -174,7 +177,10 @@ trace_metric_derive = { path = "components/trace_metric_derive" }
 trace_metric_derive_tests = { path = "components/trace_metric_derive_tests" }
 tonic = "0.8.1"
 tokio = { version = "1.29", features = ["full"] }
-wal = { path = "wal" }
+wal = { path = "src/wal" }
```

Comment:
I wonder if this is necessary to split different wal implementation to different crates, any advantages?

Reply:
It's possible we retain the code in wal crate and guard them with different feature gate. Factor out impl into dedicated crates is for:

* per https://github.com/CeresDB/ceresdb/issues/1222#issuecomment-1726786002 suggested
* exclude a crate can be straightforward than conditionally opt-out some code in the same crate, but perhaps we can narrow the opt code into the same mod.

Generally, I don't have a strong preference, either should be viable.

Reply:
OK, I also don't have strong preference, keep it as it's now.